ABIDE_I/data_convert.py

```diff
@@ -0,0 +1,40 @@
+"""ABIDE I phenotype data conversion from CSV to TSV format.
+
+This script finds the data file as downloaded from http://www.nitrc.org/frs/downloadlink.php/4912 in .csv format and
+converts it to .tsv format file to be in compliance with BIDS specification. The TSV files are deposited in the
+`phenotype/` directory along with the data dictionary in JSON format.
+"""
+
+import argparse
+from pathlib import Path
+
+import pandas as pd
+
+
+def get_args():
+    parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter, description=__doc__)
+    parser.add_argument('-i', '--input-file', type=Path, metavar='INPUT_FILE', required=True,
+                        help='Path to the phenotype file in CSV format as downloaded from ABIDE I webpage.')
+    parser.add_argument('-o', '--output-dir', type=Path, metavar='OUTPUT_DIR', required=False,
+                        default=Path(__file__).parent, help='Path to output BIDS formatted directory.')
```

Comment:
Changing them to positional arguments for intended future behavior. We can base the final root-level CLI off this CLI.

```suggestion
    parser.add_argument('input', type=Path, metavar='INPUT_FILE', required=True,
                        help='Path to the phenotype file in CSV format as downloaded from ABIDE I webpage.')
    parser.add_argument('output', type=Path, metavar='OUTPUT_DIR', required=False,
                        default=Path(__file__).parent, help='Path to output BIDS formatted directory.')
```